src/Stoa.ts

```diff
@@ -21,11 +23,41 @@ class Stoa {
             res.send(req.params.address + ':' + req.query.height);
         });
 
+        /**
+         * When a request is received through agora-history push
```

Comment:
```suggestion
         * When a request is received through the `/push` handler
```

`agora-history` is just a tool for you to test.

Reply:
Apply the suggestion.